proto/tendermint/consensus/types.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+package tendermint.consensus;
+
+option go_package = "github.com/lazyledger/lazyledger-core/proto/tendermint/consensus";
+
+import "gogoproto/gogo.proto";
+import "tendermint/types/types.proto";
+import "tendermint/libs/bits/types.proto";
+
+// NewRoundStep is sent for every step taken in the ConsensusState.
+// For every height/round/step transition
+message NewRoundStep {
+  int64  height                   = 1;
+  int32  round                    = 2;
+  uint32 step                     = 3;
+  int64  seconds_since_start_time = 4;
+  int32  last_commit_round        = 5;
+}
```

Comment:
Does this go through the ABCI boundary? I don't think rounds are necessary in optimint.